docs/Language.md

```diff
@@ -0,0 +1,64 @@
+    * **Me/My**: When the user is interacting with the product on things like buttons- ONLY if absolutely necessary for clarity. _First person is not preferred._
+
+* **Dates**
+    * **Do**: Always respect the user's shorthand date format (DD/MM/YY, MM/DD/YY, YY/MM/DD, etc.)
+    * **Do**: Use `d MMM YYYY` (eg: 7 Jul 1977) in contexts where a more conversational date is required
+
+* **Use input agnostic language** where possible instead of input specific terminology
+    * **Do**: Pick your preferred email address
+    * **Don't**: Click on your preferred email address
+
+* **Action words should follow the following rules** 
+    * **Create/Delete**: When a user will be creating/deleting a system object like a Self-Service Action, Pulse, or Person
+    * **Add/Remove**: When a user is adding/removing additional data to a system object like a Phone Number to a Person, or Measurements to a Goal
+    * **Update**: When a user is editing a system object
+    * **Exceptions**: Occasionally there will be more context appropriate action words such as `Set Goal` and `Issue Task` which are preferred to the above. The appropriate destructive action word should still be used.
+    * **Page Titles and Submit Buttons**: For titles and buttons, use the following format `[Action Name] [System Object Name]`. For example: `Create Person`, `Add Phone Number`, `Set Goal`. On edit forms the submit button should always be `Save`
+    * **Don't**: Use the word `new` on creation actions. It is implied.
+
+## Capitalization and Punctuation
+* **Use Title Case for buttons, page titles, menu items, and labels** - most words are capitalized
+    * **Example**: Set a Goal
+    * **Do Capitalize**: The first letter of nouns, adjectives, verbs, adverbs, and pronouns
+    * **Do Capitalize**: The first and last words of the text element
+    * **Do Capitalize**: The second word of a hyphenated compound unless it is an article or preposition (for example: “Quick-Search Options” and “Add-on Choices”).
+    * **Don't Capitalize**: Articles (“a,” “an,” “the”) or prepositions with fewer than five letters (“to,” “on,” “at,” “by,” “from”) unless they’re the first or last words of the text element.
+
+* **Use Sentence Case for descriptions, help text, and anything conversational** - most words are lowercase
+    * **Example**: You haven't completed any training in the last quarter.
+    * **Do**: Only capitalize the first word of the text element, as well as any proper nouns.
+
+* **Explain acronyms at the first instance on the page** - when using acronyms
+    * **Example**: Performance Improvement Plan (PIP)
+
+* **Use serial (Oxford) commas** for clarity in lists
+    * **Do**: The Performance Improvement page features all of your skills, goals, and training records
+    * **Don't**: The Performance Improvement page features all of your skills, goals, and training records
```

Comment:
you did the same thing in the Do and Don't here

Reply:
just dont use lists

Reply:
or do

Reply:
LOL Good catch